src/components/UI/Header/index.tsx

```diff
@@ -74,12 +99,56 @@ export default function Header({
                 haveDownloads > 0 ? `(${haveDownloads})` : ''
               }`}
             </span>
+          </span>
+        )}
+        {handleFilter && category==='unreal' && (
+          <span className="selectFilter">
+            <span>{t('Filter')}:</span>
             <span
               className={filter === 'unreal' ? 'selected' : ''}
               onClick={() => handleFilter('unreal')}
             >
-              {t('Unreal Marketplace')}
+              {t('All')}
+            </span>
+            <span
+              className={filter === 'asset' ? 'selected' : ''}
+              onClick={() => handleFilter('asset')}
+            >
+              {t('Assets')}
             </span>
+            <span
+              className={filter === 'plugin' ? 'selected' : ''}
+              onClick={() => handleFilter('plugin')}
+            >
+              {t('Plugins')}
+            </span>
+            <span
+              className={filter === 'project' ? 'selected' : ''}
+              onClick={() => handleFilter('project')}
+            >
+              {t('Projects')}
+            </span>
+            <select
+              className={filter.includes('UE_') ? 'selected' : ''}
+              onChange={(event) => handleFilter(event.target.value)}
+            >
+              <option value='UE_4.26'>4.26</option>
```

Comment:
I guess one improvement would be having this list of versions on an array and then just map it here. Because in the future can be easier to add/remove versions.